streams/stream.proto

```diff
@@ -24,19 +19,14 @@ package saga.rpc.streams;
 message StatusUpdate {
     string trace_id = 1;
     oneof status_update {
-        streams.nftbridge.NftBridgeUpdate nft_bridge_update = 2;
-        streams.currency.CurrencyUpdate currency_update = 3;
-        streams.item.ItemUpdate item_update = 4;
-        streams.itemtype.ItemTypeUpdate item_type_update = 5;
-        streams.listing.ListingUpdate listing_update = 6;
-        streams.myth.MythTokenUpdate myth_token_update = 7;
-        streams.offer.OfferUpdate offer_update = 8;
-        streams.order.OrderUpdate order_update = 9;
-        streams.payment.PaymentUpdate payment_update = 10;
-        streams.playerwallet.PlayerWalletUpdate player_wallet_update = 11;
-        streams.reservation.ReservationUpdate reservation_update = 12;
+        streams.currency.CurrencyUpdate currency_update = 2;
+        streams.item.ItemUpdate item_update = 3;
+        streams.itemtype.ItemTypeUpdate item_type_update = 4;
+        streams.myth.MythTokenUpdate myth_token_update = 5;
+        streams.playerwallet.PlayerWalletUpdate player_wallet_update = 6;
+        streams.reservation.ReservationUpdate reservation_update = 7;
     }
-    google.protobuf.Timestamp created_at = 15;
+    google.protobuf.Timestamp created_at = 8;
```

Comment:
Really should retain the numbers, not change them so we are backward compatible.  Need to get in the habit of that now that we are going into prod mode.

Reply:
Okay, i though about that but because we are not in prod yet i though it was OK. I can revert it. I'll include this flag as well. ie: 

`google.protobuf.Timestamp created_at = 15 [deprecated=true]`

Reply:
Yeah you are right since not in prod yet it would be okay, but we should start doing it now.

Reply:
That will require the all the stream directories (payment, order, offer, listing, nftbridge) i removed to be added back in so the imports don't break. Do you want that?